src/DMXSerial2.cpp

```diff
@@ -364,7 +378,8 @@ void DMXSerialClass2::init(struct RDMINIT *initData, RDMCallbackFunction func, R
   } else {
     // set default values
     _startAddress = 1;
-    strcpy (deviceLabel, "new");
+    _personalityNumber = _initData->defaultPersonalityNumber;
+    strncpy(deviceLabel, "new", DMXSERIAL_MAX_RDM_STRING_LENGTH);
```

Comment:
strncpy not required with the fixed length string.

Reply:
Agreed, I assume the compiler will optimise it out. I'm also trying to safety proof it from someone going "I want this to be a longer string" and overflowing.